src/main/webapp/WEB-INF/jsp/coda.jspf

```diff
@@ -51,7 +51,7 @@
 	<script src="/assets/js/deps/Markdown.Sanitizer.js"></script>
 	<script src="/assets/js/deps/Markdown.Editor.js"></script>
 	<script src="/assets/js/deps/marked.js"></script>
-	<script src="/assets/js/deps/filepicker.js"></script>
+	<%--<script src="/assets/js/deps/filepicker.js"></script>--%>
```

Comment:
No, it wasn't! I've fixed it directly on master.

Em seg, 27 de jul de 2015 21:50, Leonardo Cesar Wolter <
notifications@github.com> escreveu:

> In src/main/webapp/WEB-INF/jsp/coda.jspf
> <https://github.com/caelum/mamute/pull/184#discussion_r35605134>:
>
> > @@ -51,7 +51,7 @@
> >  	<script src="/assets/js/deps/Markdown.Sanitizer.js"></script>
> >  	<script src="/assets/js/deps/Markdown.Editor.js"></script>
> >  	<script src="/assets/js/deps/marked.js"></script>
> > -	<script src="/assets/js/deps/filepicker.js"></script>
> > +	<%--<script src="/assets/js/deps/filepicker.js"></script>--%>
>
> Is this on purpose?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/caelum/mamute/pull/184/files#r35605134>.
>